airbyte-integrations/connectors/source-kafka/src/main/java/io/airbyte/integrations/source/kafka/format/AbstractFormat.java

```diff
@@ -0,0 +1,87 @@
+package io.airbyte.integrations.source.kafka.format;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.integrations.source.kafka.KafkaProtocol;
+import org.apache.kafka.clients.CommonClientConfigs;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.common.config.SaslConfigs;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public abstract class AbstractFormat implements KafkaFormat {
+    private static final Logger LOGGER = LoggerFactory.getLogger(AbstractFormat.class);
+
+    protected Set<String> topicsToSubscribe;
+    protected JsonNode config;
+
+    public  AbstractFormat(JsonNode config) {
+        this.config=config;
+
+    }
+
+    protected abstract KafkaConsumer<String, ?> getConsumer();
+
+
```

Comment:
@sivankumar86 Lots of whitespace have you ran `./gradlew format`

Reply:
it seems, there is no task. Task 'format' not found in project ':airbyte-integrations:connectors:source-kafka'.

Reply:
Done